atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -86,11 +90,21 @@ private void handleMultipleTimestampFetch() {
         }
     }
 
-    private String saveThreadDumps() throws IOException {
-        File file = File.createTempFile("atlas-timestamps-log", ".log");
+    @VisibleForTesting
```

Comment:
Self-review comment: I wanted to test this, but didn't want to expose a new public method to do it, and I also didn't want to trigger it through a drawn-out process and then have to parse the logs to divine the temp file location. So I slapped `@VisibleForTesting` on it and tested this method.

This could imply a SRP violation - is it worth extracting a class whose single responsibility is to give you a temporary file when you supply the filename?